tools/kebab-migration.toit

```diff
@@ -83,15 +87,37 @@ main args:
 migrate parsed/cli.Parsed:
   toitc := parsed["toitc"]
   sources := parsed["source"]
+  abort_on_error := parsed["abort-on-error"]
 
   if not toitc:
     toitc = find_toitc_from_jag
 
   check_toitc_version toitc
 
   migration_points := []
-  sources.do:
-    out := pipe.backticks toitc "-Xmigrate_dash_ids" "--analyze" it
+  sources.do: | source/string |
+    print "converting $source"
```

Comment:
This printing is a little bit inconsistent with the printing in the other parts of this program (starts with lower case). Maybe we should fix this before we ship a new SDK with this bundled?